components/builder-depot/src/server.rs

```diff
@@ -1241,6 +1241,73 @@ fn list_package_versions(req: &mut Request) -> IronResult<Response> {
     }
 }
 
+fn list_origin_packages_for_version(req: &mut Request) -> IronResult<Response> {
+
+    let (start, stop) = match extract_pagination(req) {
+        Ok(range) => range,
+        Err(response) => return Ok(response),
+    };
+
+    let ident = {
+        let params = req.extensions.get::<Router>().unwrap();
+        ident_from_params(params).to_string()
+    };
+
+    let packages: RouteResult<OriginPackagesForVersionResponse>;
+    let mut request = OriginPackagesForVersionRequest::new();
+    request.set_start(start as u64);
+    request.set_stop(stop as u64);
+
+    request.set_ident(OriginPackageIdent::from_str(ident.as_str()).expect(
```

Comment:
Yeah, I think I'm just going to remove it, as I don't think it's doing anything useful here, and shouldn't panic given the way the route gets matched and parsed.